app/scripts/views/components/barChart/barChart.js

```diff
@@ -131,8 +154,14 @@
           _returnValue = 'NA';
         }
       } else if (v === opts_.xDomain[0]) {
+        if (data_.smallDataFlag) {
+          return '<=' + e(opts_.xDomain[1]);
+        }
         return '<=' + opts_.xDomain[1];
       } else if (v === opts_.xDomain[opts_.xDomain.length - 2]) {
+        if (data_.smallDataFlag) {
```

Comment:
You still need the if condition, but parts `return '>' + ` and `opts_.xDomain[opts_.xDomain.length - 3]` are the same